File: Go/Slice/slice-contains.go
package main

import (
	"fmt"
)

func main() {
	// Create a slice of strings
	randomSlice := []string{"A", "B", "C", "D", "E", "F", "G", "H", "I", "J"}
	// Check if the slice contains a value
	fmt.Println(sliceContains(randomSlice, "A"))
}

// Check if the slice contains a value and return a bool.
func sliceContains(slice []string, cointains string) bool {
	for _, value := range slice {
		if value == cointains {
			return true
		}
	}
	return false
}

File: Python/Slice/slice_contains.py
# Determine whether or not the array includes a specific element.
def slice_contains(provided_slice, provided_element):
    return provided_element in provided_slice

def main():
    # Make a slice using a random set of data.
    randomSlice = ["c", "a", "b"]
    # Determine whether or not the array includes a specific element.
    print(slice_contains(randomSlice, "a"))

main()
